exchanges/account/account.go

```diff
@@ -19,10 +19,13 @@ func init() {
 }
 
 var (
-	errHoldingsIsNil                = errors.New("holdings cannot be nil")
-	errExchangeNameUnset            = errors.New("exchange name unset")
-	errExchangeHoldingsNotFound     = errors.New("exchange holdings not found")
-	errAssetHoldingsNotFound        = errors.New("asset holdings not found")
+	// ErrExchangeHoldingsNotFound is a common error for when an exchange
+	// holdings cannot be found
+	ErrExchangeHoldingsNotFound = errors.New("exchange holdings not found")
```

Comment:
Don't think this comment helps at all.
Propose moving public errors to a block with a comment to avoid redundant comments.
```
// Public errors
var (
   ErrPengiuins = errors.New("")
)